AssertionError: `base_name` argument not specified

I am not sure if this is a bug, or just a documentation issue, but in the quickstart guide it states:
> For trivial cases you can simply set a model attribute on the ViewSet class and the serializer and queryset will be automatically generated for you. Setting the queryset and/or serializer_class attributes gives you more explicit control of the API behaviour, and is the recommended style for most applications.

When I tried this using 3.1.0 however I get the following error:
```
AssertionError: `base_name` argument not specified, and could not automatically determine the name from the viewset, as it does not have a `.queryset` attribute.
```

I had another project which is still using 2.3.8 and when I tried it there it worked fine.

In order to verify I didn't do anything wrong, I took the users examples from the [quickstart](http://www.django-rest-framework.org/tutorial/quickstart/#views) and turned it into a "trivial" example:
```
class UserViewSet(viewsets.ModelViewSet):
    """
    API endpoint that allows users to be viewed or edited.
    """
    model = User
    # queryset = User.objects.all()
    # serializer_class = UserSerializer
```
The error is very informative so kudos for that. Additionally the fix is pretty easy.  Since this works on 2.3.8 I wonder if that quote is left over from the older version, or if it should be refined to explain that in those simple cases you'll need a basename on the router entry?

I don't consider this to be a major issue at all (especially given the ease of a fix given the very useful error message), but I thought I'd open this to help clarify the quickstart guide a little.

--- a/docs/tutorial/quickstart.md
+++ b/docs/tutorial/quickstart.md
@@ -83,8 +83,6 @@ Rather than write multiple views we're grouping together all the common behavior
 
 We can easily break these down into individual views if we need to, but using viewsets keeps the view logic nicely organized as well as being very concise.
 
-For trivial cases you can simply set a `model` attribute on the `ViewSet` class and the serializer and queryset will be automatically generated for you.  Setting the `queryset` and/or `serializer_class` attributes gives you more explicit control of the API behaviour, and is the recommended style for most applications.
-
 ## URLs
 
 Okay, now let's wire up the API URLs.  On to `tutorial/urls.py`...

